src/test/java/uk/gov/hmcts/reform/managecase/api/controller/CaseAssignmentControllerTest.java

```diff
@@ -126,7 +126,7 @@ void shouldFailWithBadRequestWhenCaseIdIsNull() throws Exception {
             .content(objectMapper.writeValueAsString(request)))
             .andExpect(status().isBadRequest())
             .andExpect(jsonPath("$.errors", hasSize(1)))
-            .andExpect(jsonPath("$.errors", hasItem("Case ID can not be empty")));
+            .andExpect(jsonPath("$.errors", hasItem("Case ID has to be a valid 16-digit Luhn number")));
```

Comment:
NB: If you do expend the validation (see comments in domain class) then you may need to expand the tests to cover them.